jax/lax/lax.py

```diff
@@ -4256,7 +4219,22 @@ def _dilate_shape(shape, dilation):
   return onp.where(shape == 0, 0,
                    onp.multiply(dilation, onp.subtract(shape, 1)) + 1)
 
+def is_shape_nonnegative(shape):
```

Comment:
Could you prefix the name of this function and the other similar utility functions with an underscore?

Reply:
Sure.